internal/archiver/tree.go

```diff
@@ -195,26 +195,13 @@ func (t *Tree) add(fs fs.FS, target, root string, pc []string) error {
 	return nil
 }
 
-func (t Tree) String() string {
```

Comment:
This is not dead code. The String method is used via the `debug.Log` calls in `NewTree` (last function in tree)

Reply:
Ok, restored.

Reply:
tree_test.go now fails to compile when this method is not present.